### PR TITLE
Migrate Strapi version to 3.4.1 (latest)

### PR DESCRIPTION
**Pull request by Mcastres**

### CODE REVIEW

File: api/.strapi-updater.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (alexandrebodin) [inline api/.strapi-updater.json:1]: can you add this file to the gitignore :)

**Review by alexandrebodin — COMMENTED** [review]
LTM, just need to remove the update file

File: api/.strapi-updater.json (hunk 1 above)
Comment (Mcastres) [reply api/.strapi-updater.json]: Done

**Review by alexandrebodin — APPROVED** [review]
LGTM

**Pull request merged**